servconf.c

```diff
@@ -568,7 +570,7 @@ typedef enum {
 	sPerSourcePenalties, sPerSourcePenaltyExemptList,
 	sClientAliveInterval, sClientAliveCountMax, sAuthorizedKeysFile,
 	sGssAuthentication, sGssCleanupCreds, sGssStrictAcceptor,
-	sAcceptEnv, sSetEnv, sPermitTunnel,
+	sAcceptEnv, sSetEnv, sPermitTunnel, sTunnelOptions,
```

Comment:
I'm not sure we want to add a custom directive to `sshd_config` as this would be a divergence from the upstream code. Can the existing `ethernet` argument  for the `sPermitTunnel` directive be used?

Reply:
This new option is not a must for me. I added it because I was missing a way to make sure tu use only a subset of available TAP instances. My idea was to distingush TAPs by a fixed prefix in their friendly name. On my PC there are many TAP instances I wouldn't touch because they are (supposed to be) reserved to VPNs, teleservice programs etc... I even considered to use some weird tricks based on TunnelDevice values and a numeric suffix in the name (e.g. numbers > 10000 are sname suffixes...). Please note that I added the same option to client too. 
Are you suggesting something like "PermitTunnel=ethernet[:options...]"? It looks good, bat valid only for server.

Reply:
By the way, is it possible to change (edit, add, delete files) an existing pull request, or is it better me to submit a new one? (Sorry, I am quite new in github)

Reply:
Yes - for the client side, I think the `Tunnel` and `TunnelDevice` directives can be used.

As far as GitHub, you can make changes directly to this pull request by pushing the changes to this existing branch!

Reply:
I changed it as we agreed. Bye